packages/hostScriptsServices/hostScripts/sensors.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Execute the first command
+output=$(paste <(cat /sys/class/thermal/thermal_zone*/type 2>/dev/null) <(cat /sys/class/thermal/thermal_zone*/temp 2>/dev/null) 2>/dev/null | column -s $'\t' -t | sed 's/\(.\)..$/.\1/')
```

Comment:
You should create variables to make the code cleaner. For example, you could create `THERMAL_ZONE_DIRS=/sys/class/thermal/thermal_zone*`